pop-cli-for-smart-contracts/guides/securely-sign-transactions-from-cli.md

```diff
@@ -0,0 +1,55 @@
+# Securely Sign Transactions from CLI
+Pop CLI provides an option to securely sign transactions from the CLI. It does this by opening a signing portal allowing you
+to sign the transactions using your browser extension wallet.
+
+## Example Usage: Deploying a Contract
+For a full guide on deploying a contract, see the [Deploy Your Contract Locally](./deploy-your-contract-locally.md) guide.
+
+Normally, you would provide `--suri=<private-key>` to deploy the contract. However, this should only be used for development accounts.
```

Comment:
I feel we should append something at the end here mentioning that this implies insecure handling of secretes.

Reply:
https://github.com/r0gue-io/pop-docs/pull/17